tests/e2e/utils/workspace/IUrlResolver.ts

```diff
@@ -0,0 +1,13 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+export interface IApiUrlResolver {
```

Comment:
just one tiny detail, maybe it's a good idea if the class/interface matches the file name